armi/reactor/grids/grid.py

```diff
@@ -90,6 +99,21 @@ def symmetry(self) -> str:
         .. impl:: Grids shall be able to repesent 1/3 and full core symmetries.
             :id: I_ARMI_GRID_SYMMETRY
```

Comment:
```suggestion
            :id: I_ARMI_GRID_SYMMETRY0
```
optional...I see a `1` down below